### ISSUE TITLE
Checklist: August 17–23

### ISSUE THREAD

**Issue by billyroh** [wafflejs/wafflejs.github.io#48]
### Speakers & Performers
* [x] Fill 3 out of 4 speaking slots (@billyroh)
  * [x] Reach out to Isaac, Paul, James (@visnup)
  * [x] Reach out to Zach (@billyroh)
* [x] Fill 3 out of 4 performing slots
  * [ ] Reach out to ~~Myles~~, Emily (@visnup)
  * [x] Ask Natasha which spots she'd like (@visnup)
* [x] Tweet out link to https://github.com/wafflejs/wafflejs.github.io/issues/34 (@billyroh)

### Sales and Promotion
* [x] Sell 28 out of 28 of early bird tickets
* [ ] Sell 20 out of 51 regular tickets
* [ ] Get two or more sponsors ($500 each)
  * [x] Opendoor (@billyroh)
  * [ ] Particle (@emilyrose)
  * [ ] Twitter (@kellyk)
  * [ ] Crunchyroll (@kellyk)
* [x] Give out discount code to each of the speakers (@billyroh)
* [ ] Ask BrooklynJS peeps to tweet out link (@visnup)

### Logistics
* [x] Buy 6000-lumen projector (@visnup)
* [x] Check with Golden Waffle to see if bulk orders are possible (@billyroh)
* [x] Order more stickers (@visnup)
* [ ] Get two A/V people (@visnup)
* [x] Start accounting spreadsheet (@billyroh)
* [ ] Close out last month's accounting spreadsheet (@visnup)
* [x] Invite Kelly to the Twitter account (@billyroh)

**Comment by kellyk** [comment]
Re: sponsorship
 * What do sponsors get? (ie. logo placement, announcement slot, etc.)
 * Is this a one-time(ish) fundraising, or do we need $1k in donations every month?
 * Do we have any existing e-mail template or a site with sponsorship packages available?

**Comment by billyroh** [comment]
@kellyk Good questions!
> What do sponsors get? (ie. logo placement, announcement slot, etc.)
For the August event, we did the following:
* Mentioned the sponsors in the emails sent out to attendees
* Hosts talked about the sponsors in between each talk
I'm totally down to do it differently this time though.

> Is this a one-time(ish) fundraising, or do we need $1k in donations every month?
Yeah, we need $1k+ in donations every month to cover the cost of the venue, A/V, etc. I just invited you to the accounting spreadsheet if you wanna check it out :smile: 

> Do we have any existing e-mail template or a site with sponsorship packages available?
Nope, but we should! Are there any good examples of that...?
